### PR TITLE
Create loverboy-ent

### PR DESCRIPTION
**Pull request by Skyy09**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Comment by Skyy09** [comment]
I made sure it's in /domains/

File: domains/loverboy-ent
Comment (CuteDog5695) [inline domains/loverboy-ent:1]: File needs to be in json format.

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/loverboy-ent
Comment (Skyy09) [reply domains/loverboy-ent:1]: It is

Comment (Skyy09) [reply domains/loverboy-ent:1]: {
  "owner": {
    "username": "skyy09",
    "organization": "Loverboy Website Manager",
    "enterprise": "Loverboy Enterprise",
    "email": "29686@ghaps.org",
    "description": "Loverboy Enterprise is a development enterprise organization dedicated to creating programs under special circumstances for specific businesses, teams, individuals, and organizations."
  },
  "record": {
    "CNAME": "lb-ent.github.io"
  }
}

Comment (CuteDog5695) [reply domains/loverboy-ent:1]: The actual file isn't in json format. You need to add `.json` at the end of the file's name

Comment (Skyy09) [reply domains/loverboy-ent:1]: Oh!

Comment (Skyy09) [reply domains/loverboy-ent:1]: I apologize, I completely forgot about that.

Comment (Skyy09) [reply domains/loverboy-ent:1]: Finished. I apologize about that.

**Comment by Skyy09** [comment]
Are you guys going to merge it?

**Review by iostpa — CHANGES_REQUESTED** [review]
Site wasn't accessible, but I went to the sites repo and if its the correct one its not complete. I would like you to add more to your website.
I could be wrong here, so if you think I made a mistake here, please give a preview of the website.

**Comment by Skyy09** [comment]
> Site wasn't accessible, but I went to the sites repo and if its the correct one its not complete. I would like you to add more to your website. I could be wrong here, so if you think I made a mistake here, please give a preview of the website.

It's for email.

**Comment by CuteDog5695** [comment]
Then why didn't you add MX records for the main file? If I merged this right now it would only be pointed to a website, not a email service.

**Comment by Skyy09** [comment]
Done! <3

File: domains/loverboy-ent.json [hunk 1]
```diff
@@ -3,10 +3,33 @@
...
+      },
+      {
+        "priority": 10,
+        "server": "ALT3.ASPMX.L.GOOGLE.COM"
+      },
+      {
+        "priority": 10,
+        "server": "ALT4.ASPMX.L.GOOGLE.COM"
+      }
+    ]
```
Comment (CuteDog5695) [inline domains/loverboy-ent.json:33]: wtf?
Read docs please

**Review by iostpa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**